schema.sql

```diff
@@ -7,6 +7,7 @@ CREATE TABLE animals (
     date_of_birth date,
     escape_attempts int,
     neutered boolean,
-    weight_kg real
+    weight_kg real,
+    species varchar
```

Comment:
- [ ] <b>REQUIRED:</b> You are expected to add a new column called species with a string type by using `ALTER TABLE` command, not just putting in the previous command. Kindly fix that.